### PR TITLE
Commented entry for BrickPi Input Port 5 I2C baudrate

### PR DESCRIPTION
**Pull request by bmegli**
Hmm, now I will also have to update the docs on github.io to uncomment
the line instead of adding it...

### CODE REVIEW

File: projects/ev3dev-jessie/rpi-base/root/boot/flash/config.txt [hunk 1]
```diff
@@ -27,6 +27,10 @@
 # by default the Bluetooth uses the same UART as BrickPi
 #dtoverlay=pi3-disable-bt
 
+# uncomment only if you are using BrickPi (not BrickPi+) 
+# and want to use Input Port 5 for I2C/NXT sensor
```
Comment (dlech) [inline projects/ev3dev-jessie/rpi-base/root/boot/flash/config.txt]: We could add a sentence to say that some sensors work at higher baud rates

File: projects/ev3dev-jessie/rpi-base/root/boot/flash/config.txt [hunk 2]
```diff
@@ -27,6 +27,10 @@
 # by default the Bluetooth uses the same UART as BrickPi
 #dtoverlay=pi3-disable-bt
 
+# uncomment only if you are using BrickPi (not BrickPi+) 
+# and want to use Input Port 5 for I2C/NXT sensor 
+#dtparam=i2c_baudrate=9600
```
Comment (dlech) [inline projects/ev3dev-jessie/rpi-base/root/boot/flash/config.txt]: EV3 actually uses `10000` instead of `9600`

**Comment by dlech** [comment]
FYI, these changes won't ship until we do a new image release.

File: projects/ev3dev-jessie/rpi-base/root/boot/flash/config.txt (hunk 2 above)
Comment (bmegli) [inline projects/ev3dev-jessie/rpi-base/root/boot/flash/config.txt]: Yes, you are right in both. Probably it's better to run at higher if possible.
I only have CruizCore little to test here, it works ok at 9600 and higher, but starting from some baudrate it stops working properly (no errors in dmesg but -1 on angle value for example all the time)

Mindstorms NXT Hardware Developper Kit (page 9) states that: 
> I2C communication is running at 9600 bit/s

So at least in theory I2C/NXT sensors should at least work at this (slow!) speed and were tested for it.

But yes, I will update the docs and mention the somebody may want to tweak the value.

Comment (bmegli) [inline projects/ev3dev-jessie/rpi-base/root/boot/flash/config.txt]: Do you think we should use `10000` as in EV3 as the default value?

Comment (dlech) [inline projects/ev3dev-jessie/rpi-base/root/boot/flash/config.txt]: Yes. it is at least a *tiny* bit faster.

File: projects/ev3dev-jessie/rpi-base/root/boot/flash/config.txt [hunk 3]
```diff
@@ -28,7 +28,7 @@
 #dtoverlay=pi3-disable-bt
 
 # uncomment only if you are using BrickPi (not BrickPi+) 
-# and want to use Input Port 5 for I2C/NXT sensor 
+# and want to use Input Port 5 for I2C/NXT sensor, some sensors
```
Comment (dlech) [inline projects/ev3dev-jessie/rpi-base/root/boot/flash/config.txt]: I think you missed part of the sentence.

**Comment by bmegli** [comment]
Thank you. I suppose I have not hit ctrl+s in text editor before commit

**Pull request merged**